core/src/main/java/com/linecorp/armeria/common/multipart/BodyPartHeaders.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linecorp.armeria.common.multipart;
+
+import java.util.List;
+import java.util.Map.Entry;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.common.MediaType;
+
+import io.netty.util.AsciiString;
+
+/**
+ * A body part headers.
+ */
+public interface BodyPartHeaders extends Iterable<Entry<AsciiString, String>> {
```

Comment:
Is there a reason we have to define a separate interface than `RequestHeaders`? Perhaps we can add `contentDisposition()` there instead?